src/test/scala/org/hammerlab/guacamole/commands/GermlineAssemblyCallerSuite.scala

```diff
@@ -1,34 +1,203 @@
 package org.hammerlab.guacamole.commands
 
-import org.bdgenomics.adam.rdd.ADAMContext
+import org.apache.spark.SparkContext
 import org.hammerlab.guacamole.commands.GermlineAssemblyCaller.Arguments
-import org.hammerlab.guacamole.util.{ GuacFunSuite, TestUtil }
-import org.scalatest.Matchers
+import org.hammerlab.guacamole.reads.Read
+import org.hammerlab.guacamole.reference.ReferenceBroadcast
+import org.hammerlab.guacamole.util.TestUtil
+import org.hammerlab.guacamole.variants.CalledAllele
+import org.hammerlab.guacamole._
+import org.scalatest.{ BeforeAndAfter, FunSuite, Matchers }
 
-class GermlineAssemblyCallerSuite extends GuacFunSuite with Matchers {
+class GermlineAssemblyCallerSuite extends FunSuite with Matchers with BeforeAndAfter {
 
-  sparkTest("test assembly caller: illumina platinum tests") {
-    val input = "illumina-platinum-na12878/NA12878.10k_variants.plus_chr1_3M-3.1M.bam"
+  val args = new Arguments
 
-    val args = new Arguments
-    args.reads = TestUtil.testDataPath(input)
-    args.loci = "chr1:772754-772755"
-    args.referenceFastaPath = TestUtil.testDataPath("illumina-platinum-na12878/chr1.prefix.fa")
-    args.parallelism = 2
-    args.variantOutput = TestUtil.tmpFileName(suffix = ".vcf")
-    args.snvWindowRange = 50
-    args.kmerSize = 45
+  val input = "illumina-platinum-na12878/NA12878.10k_variants.plus_chr1_3M-3.1M.bam"
+  args.reads = TestUtil.testDataPath(input)
+  args.loci = "chr1:772754-772755"
+  args.parallelism = 2
 
-    GermlineAssemblyCaller.Caller.run(args, sc)
+  var sc: SparkContext = _
+  var reference: ReferenceBroadcast = _
+  var readSet: ReadSet = _
 
-    val variants = new ADAMContext(sc).loadVariants(args.variantOutput).collect()
+  before {
+    sc = Common.createSparkContext()
```

Comment:
I was struggling with this recently; surprised that `sparkTest`, `sparkBefore` etc. each set up and tear down a SparkContext! Will have to use this pattern more often